.github/workflows/version.yml

```diff
@@ -0,0 +1,23 @@
+name: version
+on:
+  release:
+    types:
+      - created
```

Comment:
I was thinking about this... I think there is a slight issue. We need to update the version BEFORE the release is created and this needs to be done before creating the tag since the release is generated based on the tag.

With this current flow:
1. tag is created 1.0.2
2. release is created  in draft (not published)
3. this workflow kicks off updates this field to 1.0.2
4. release is published but contains previous version value